### ISSUE TITLE
Clicking in the range of the menu bar drag the window in GTK based app

### ISSUE THREAD

**Issue by KDB2** [linuxmint/muffin#191]
Dear team,

I have noticed that in some application (like Nemo), clicking (with mouse button release) in the vertical range of the menu bar selects the window (drag feature) despite the release of the mouse button.

Clicking in other parts of the windows is working correctly.

Regards,
KDB

**Comment by JosephMcc** [comment]
I assume you are running betsy? I've encountered this as well. Doesn't happen under Mint (gtk3.10)

**Comment by KDB2** [comment]
Hello @JosephMcc,

I have Betsy that's right. 
KDB

**Comment by monsta** [comment]
Looks similar to what I see in MATE - https://github.com/mate-desktop/marco/issues/98 (skip the part about window jumping - it seems to be a multi-monitor issue). Seems like a GTK+ 3.14 issue to me.

**Comment by JosephMcc** [comment]
@monsta I just discovered this was issue in Mate in Betsy a couple days ago so you're probably right.

**Comment by monsta** [comment]
So I've tested it also in plain Debian Jessie with stock versions of MATE and Xfce. Tested it with galculator which is GTK+3 app. Tried xfwm4, marco and metacity. The bug is apparently in GTK+ 3.14, as it happens with any of these WMs and any GTK+ theme (even Adwaita).

Well, given the usual Debian Stable update policy, they probably won't apply the fix even if I'll find it and send them a patch. Annoying UI bugs usually aren't considered serious enough there. But we have a chance to fix this in Betsy - just need to find the fix itself :smile:
